how-it-works/6_webaccess/02_asset_certification.subpage.md

```diff
@@ -33,3 +33,7 @@ IC-native browser or a web-extension.
 
 
 For more information on certification, check [Certified Variables](/how-it-works/response-certification/).
+
+[Asset Certification Wiki Article](https://wiki.internetcomputer.org/wiki/HTTP_asset_certification)
+
+[Rust Canister Development Security Best Practices](https://internetcomputer.org/docs/current/references/security/rust-canister-development-security-best-practices#asset-certification)
```

Comment:
```suggestion
[Rust Canister Development Security Best Practices](/docs/current/references/security/rust-canister-development-security-best-practices#asset-certification)
```
relative links please, otherwise CI won't catch if it breaks